src/util/defs.h

```diff
@@ -4,7 +4,7 @@
 
 // Maximum buffer length to each EngineObject::process call.
 //TODO: Replace this with mixxx::AudioParameters::bufferSize()
-constexpr unsigned int MAX_BUFFER_LEN = 160000;
+constexpr unsigned int MAX_BUFFER_LEN = 32 * 1024;
```

Comment:
Please add a comment that describes the considerations for the new value.

Reply:
The thing is that this value is arbitrary.
In zulip, I've read that mix doesn't support processing blocks larger than 4K.
So I'd actually even shrink it down to MAX_FRAMER * MAX_CHANNELS (4096 * 2).

Reply:
I've added a comment.
I think in the long run this constant should be gone.